src/node_options.cc

```diff
@@ -107,6 +107,11 @@ void PerIsolateOptions::CheckOptions(std::vector<std::string>* errors) {
         "--experimental-report is set");
   }
 
+  if (report_compact) {
```

Comment:
This can be removed now (but will probably be needed again when this is backported).